src/careamics/dataset/patching/patching.py

```diff
@@ -20,21 +20,26 @@ class Stats:
     means: Union[np.ndarray, tuple, list, None]
     stds: Union[np.ndarray, tuple, list, None]
 
+    def get_statistics(self) -> tuple[list[float], list[float]]:
+        """Return the means and standard deviations.
 
-@dataclass
-class PatchedOutput:
-    """Dataclass to store patches and statistics."""
+        Returns
+        -------
+        tuple of two lists of floats
+            Means and standard deviations.
```

Comment:
Again documentation, this is means per channel?

Reply:
All of this changed in the Welford PR, I made now another pass to the docstrings.